### PR TITLE
style: update event card color

### LINKED ISSUE #110: Create a event card to be used on homepage calendar
### TODO
Need to create a event card for homepage calendar

### Reference
image

### PR DESCRIPTION
**Pull request by iRohitGaur**
### What is the change?
update the color of event card

### Is it bug?
No

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [ ] Web

#### Browsers

- [ ] Chrome
- [ ] Safari
- [ ] Firefox

### Before / After Change Screenshots
Screenshot 2023-04-13 at 19 55 58

> For visual or interaction changes. Can be video / screenshot.
closes #110

### CODE REVIEW

File: app/styles/global.css [hunk 1]
```diff
@@ -1,3 +1,29 @@
 /* This is a global CSS file  */
 @import url('react-big-calendar/lib/css/react-big-calendar.css');
 @import url('react-big-calendar/lib/addons/dragAndDrop/styles.css');
+
+/* overriding react-big-calendar's css properties */
+.rbc-event {
+  background-color: #e7e5e4;
```
Comment (harshith-venkatesh) [inline app/styles/global.css:7]: Is this through the pallete that was softly agreed upon in the discussion.
Was it discussed with @yesyash

**Review by harshith-venkatesh — APPROVED** [review]
Please address my comment, rest looks good @iRohitGaur

File: app/styles/global.css (hunk 1 above)
Comment (iRohitGaur) [reply app/styles/global.css:7]: @harshith-venkatesh the pallete is still pending. For now I have used the hex colors from the Figma file. Later when we have the pallet, we can replace the colors from the whole codebase.

Comment (harshith-venkatesh) [reply app/styles/global.css:7]: Thanks for the update @iRohitGaur

**Pull request merged**